File: CPP/Program/Product.cpp
#include <iostream>
#include <string>
using namespace std;

class Product {
protected:
    string idProduct;
    string name;
    string brand;
    double price;

public:
    void setId(string id) {
        idProduct = id;
    }

    void setName(string n) {
        name = n;
    }

    void setBrand(string b) {
        brand = b;
    }

    void setPrice(double p) {
        price = p;
    }

    string getId() {
        return idProduct;
    }

    string getName() {
        return name;
    }

    string getBrand() {
        return brand;
    }

    double getPrice() {
        return price;
    }
};

File: Python/Program/product.py
class Product:
    #constructor
    def __init__(self):
        #inisialisasi atribut untuk id, nama, nama brand dan harga
        self.id = ""
        self.name = ""
        self.brand = ""
        self.price = 0.0
    
    #membuat getter dan setter
    #set Id
    def setId(self, id):
        self.id = id
    
    #get Id
    def getId(self):
        return self.id
    
    #set Name
    def setName(self, name):
        self.name = name
    
    #get Name
    def getName(self):
        return self.name
    
    #set Brand
    def setBrand(self, brand):
        self.brand = brand
    
    #get Brand
    def getBrand(self):
        return self.brand
    
    #set Price
    def setPrice(self, price):
        self.price = price
    
    #get Price
    def getPrice(self):
        return self.price
